Lib/textwrap.py

```diff
@@ -304,8 +304,12 @@ def _wrap_chunks(self, chunks):
 
             # If the last chunk on this line is all whitespace, drop it.
             if self.drop_whitespace and cur_line and cur_line[-1].strip() == '':
-                cur_len -= len(cur_line[-1])
-                del cur_line[-1]
+                # If this is the first line keep it if a non-whitespace chunk follows
+                if not lines and len(cur_line) == 1 and chunks and chunks[-1].strip() != '':
+                    pass
+                else:
+                    cur_len -= len(cur_line[-1])
+                    del cur_line[-1]
```

Comment:
```suggestion
                # Keep first non-whitespace chunk on the first line
                if not (not lines and len(cur_line) == 1 and chunks and chunks[-1].strip()):
                    cur_len -= len(cur_line[-1])
                    del cur_line[-1]
```
Maybe this is better?

Reply:
Or
```python
                if lines or len(cur_line) != 1 or not chunks or not chunks[-1].strip():
                    cur_len -= len(cur_line[-1])
                    del cur_line[-1]
```